qutebrowser/browser/downloads.py

```diff
@@ -745,20 +733,42 @@ def fetch(self, reply, fileobj=None, filename=None, auto_remove=False):
         return download
 
     @cmdutils.register(instance='download-manager', scope='window')
-    def cancel_download(self, count: {'special': 'count'}=1):
-        """Cancel the first/[count]th download.
+    def download_cancel(self, count: {'special': 'count'}=0):
+        """Cancel the last/[count]th download.
 
         Args:
             count: The index of the download to cancel.
         """
-        if count == 0:
-            return
         try:
             download = self.downloads[count - 1]
         except IndexError:
             raise cmdexc.CommandError("There's no download {}!".format(count))
         download.cancel()
 
+    @cmdutils.register(instance='download-manager', scope='window',
+                       deprecated="Use :download instead.")
+    def cancel_download(self, count: {'special': 'count'}=1):
+        """Cancel the first/[count]th download.
+
+        Args:
+            count: The index of the download to cancel.
+        """
+        self.download_cancel(count)
+
+    @cmdutils.register(instance='download-manager', scope='window')
+    def download_open(self, count: {'special': 'count'}=0):
+        """Open the last/[count]th finished download.
+
+        Args:
+            count: The index of the download to cancel.
+        """
+        finished_items = [d for d in self.downloads if d.done and d.successful]
+        try:
+            download = finished_items[count - 1]
+        except IndexError:
+            raise cmdexc.CommandError("There's no download {}!".format(count))
```

Comment:
All those error messages printing `count` could probably be improved when count is `0`, probably they should only print *There's no dowload!* in that case (*There's no download 0!* is rather confusing).